tests/tablenavigation.js

```diff
@@ -0,0 +1,2277 @@
+import { getCode } from '@ckeditor/ckeditor5-utils/src/keyboard';
+import { getData as getModelData, setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
```

Comment:
We try to follow a convention in which named imports usually land after default imports. 

Also, we try (at least to some extend) to group imports (external packages in one group (image, horizontal line), in-package imports (table, table editing) in another, global helpers (`global`, `env`) in another).

This should probably be explained in https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/contributing/code-style.html (cc @Reinmar).